crates/chia-consensus/src/spendbundle_validation.rs

```diff
@@ -33,3 +33,41 @@ pub fn get_flags_for_height_and_constants(height: u32, constants: &ConsensusCons
     }
     flags
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::consensus_constants::TEST_CONSTANTS;
+
+    #[test]
+    fn test_get_flags() {
```

Comment:
this test could be parameterized with 4 test cases, and be quite a lot smaller

Reply:
Moved to rstest with parameters